site-container.yml.sample

```diff
@@ -517,7 +517,7 @@
       run_once: true
       when: not ceph_status.failed
 
-- hosts: '{{ groups["grafana-server"][0] | default(groups["mgrs"][0]) | default(groups["mons"][0]) | default(omit) }}'
```

Comment:
@dsavineau I must be missing something, the more I read this line, the more I'm wondering why we ended up adding all these `| default()`

Reply:
Agree, this should only have grafana-server group present